### PR TITLE
Fixed #30601 -- Added warning in docs

### PR DESCRIPTION
**Pull request by angelicaferlin**
Proposed solution for https://code.djangoproject.com/ticket/30601

### CODE REVIEW

File: docs/topics/db/transactions.txt [hunk 1]
```diff
@@ -50,6 +50,23 @@ either all or none of the changes will be committed.
...
+    to ensure that all database changes are atomic and either succeed or fail 
+    together. However, this can cause problems with caching if your code expects 
+    changes to be visible outside of the transaction. Additionally, caching 
+    frameworks may not be designed to work with transactions and may not behave 
+    as expected. 
+
+    If your code uses caching or proactively updates data in the cache, consider 
+    disabling ATOMIC_REQUESTS to avoid unexpected behavior. Similarly, if your 
+    code assumes that database changes are visible outside of the transaction, 
+    you may need to update your code to take transactions into account.
```
Comment (felixxm) [inline docs/topics/db/transactions.txt:68]: Have you seen Simon's [suggestion](https://code.djangoproject.com/ticket/30601#comment:1)?

Comment (felixxm) [reply docs/topics/db/transactions.txt:68]: My first reaction is that this warning says the same thing multiple times without providing any feasible solution :thinking: e.g. it doesn't mention `transaction.on_commit`.

**Comment by felixxm** [comment]
@angelicaferlin Do you have time to keep working on this?

**Comment by angelicaferlin** [comment]
Sorry, I don´t have time and have deassigned myself from the ticket. Anyone interested can pick up from here!

**Comment by angelicaferlin** [comment]
> @angelicaferlin Do you have time to keep working on this?

Sorry, I don´t have time and have deassigned myself from the ticket. Anyone interested can pick up from here!

**Comment by nessita** [comment]
Closing following the message from the author, and resetting flags in the ticket so hopefully someone else picks this up.

**Pull request closed without merging**